auditlog_tests/tests.py

```diff
@@ -2164,6 +2164,19 @@ def test_log_entry_created_if_obj_strings_are_same_for_two_objs(self):
         self.assertEqual(int(log_create.changes_dict["related"][1]), one_simple.id)
         self.assertEqual(int(log_update.changes_dict["related"][1]), two_simple.id)
 
+    def test_rel_class_checked_first(self):
```

Comment:
Better to create a model as you described in the PR description and test with a real model instead of mocking